resources/views/committee/include/organised-events.blade.php

```diff
@@ -56,8 +56,10 @@
     </div>
 
 @endif
-
-@if(count($committee->pastHelpedEvents()) > 0)
+@php
```

Comment:
I know this limit is already in place in master, but it's kind of weird there is no pagination on the committee page to see more than 6 previously organised events. Especially for a committee like pLAN that organises a lot of actvities, mainly GameBreaks, it can give an unrealistic representation of the committee.

Reply:
Would be a nice feature request in a new issue :)